cmd/clients_update.go

```diff
@@ -0,0 +1,45 @@
+package cmd
+
+import (
+	"github.com/spf13/cobra"
+)
+
+// updateCmd represents the update command
+var clientsUpdateCmd = &cobra.Command{
+	Use:   "update <id>",
+	Short: "Update an OAuth 2.0 Client",
+	Long: `This command updates an OAuth 2.0 Client by its ID.
```

Comment:
Given that an update is a PUT and not a PATCH operation (it replaces the client) I think we should explicitly tell people that here. Otherwise someone might think that he/she can change the grant type only but instead everything else is deleted.

Reply:
Thanks for your review! 
I agree with your comments and I will improve the description.

What do you think of CLI supporting partial updates? With a flag or other command, for example: 
`hydra clients update client-1 --partial -g client_credentials` 
or 
`hydra clients patch client-1 -g client_credentials`
Maybe we can discuss this in another issue.